content/general/concepts/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Environment Variables'
+Description: 'Environment variables consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Unix'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+**Environment variables** consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.
+
+Environment variables are often set via the [terminal](https://www.codecademy.com/resources/docs/general/terminal) or in a script, though how they are managed and retrieved varies depending on the operating system or environment. For example, they can be used to store system configuration data or private keys for applications.
+
+## Relationship with Unix and Unix-like Variants
+
+On [Unix](https://www.codecademy.com/resources/docs/general/unix) and Unix-like variants such as [Linux](https://www.codecademy.com/resources/docs/open-source/linux), environment variables allow for configuration of the system and applications. They can contain information such as the default language for the system, which default text editor to use, and paths to executable files.
```

Comment:
```suggestion
On [Unix](https://www.codecademy.com/resources/docs/general/unix) and Unix-like variants such as [Linux](https://www.codecademy.com/resources/docs/open-source/linux), environment variables allow for configuration of the system and its applications. They can contain information such as the default language for the system, which default text editor to use, and paths to executable files.
```

Reply:
@yangc95 thank you for this comment, I updated and added in "its".